src/graphql/event/http.rs

```diff
@@ -182,16 +186,20 @@ impl HttpThreat {
         &self.inner.db_name
     }
 
-    async fn rule_id(&self) -> u32 {
-        self.inner.rule_id
+    /// The rule id of the event in string wthin the range representable
+    /// by a `u32`.
+    async fn rule_id(&self) -> StringNumber<u32> {
```

Comment:
I think `ID` is a better option.

Reply:
@sehkone 
I have a question. In the case of TIDB, since the rule ID of TidbRule is obtained, I thought it would be correct to change it to ID. However, I couldn't find the source for the `rule_id` value in HTTP. It appears that this value is simply a `u32`, so I'm wondering if it is also correct to change this value to ID. 
- Specifically, I’m asking whether it should be changed to ID because it is expressed as the ID of `rule_id`.